.azure/gpu-tests-lite.yml

```diff
@@ -96,6 +96,7 @@ jobs:
     - bash: python -m coverage run --source lightning_lite -m pytest --ignore benchmarks -v --junitxml=$(Build.StagingDirectory)/test-results.xml --durations=50
       env:
         PL_RUN_CUDA_TESTS: "1"
+        CUDA_LAUNCH_BLOCKING: "1"
```

Comment:
This should not be added. This will slow down tests and is meant for debugging only